README.adoc

```diff
@@ -292,17 +364,17 @@ whenever a code change introduces a defect. JUnit and the JAX-RS Client API prov
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Create the `FaultToleranceTest` class.#
-`src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceTest.java`
+#Create the `FaultToleranceIT` class.#
+`src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceIT.java`
 ----
 
-The [hotspot=21]`@Before` and [hotspot=27]`@After` annotations indicate that this method executes before or after the other test case.
+The [hotspot=21]`@BeforeEach` and [hotspot=27]`@AfterEach` annotations indicate that this method executes before or after the other test case.
```

Comment:
Change to:
"The `@BeforeEach` and `@AfterEach` annotations indicate that this method runs either before or after the other test case."

Reply:
Updated